gen/calc.py

```diff
@@ -335,11 +335,17 @@ def validate_config_subnet(config_name, subnet, version=IPVersion.IPv4):
                 (version == IPVersion.IPv6 and not isinstance(network, ipaddress.IPv6Network)):
             raise ValueError("IP version not match")
     except ValueError as ex:
-        err_msg = "Incorrect value for {}: {}. Only IPv{} subnets are allowed".format(
+        err_msg = "Incorrect value for `{}`: `{}`. Only IPv{} subnets are allowed".format(
             config_name, subnet, version)
         raise AssertionError(err_msg) from ex
 
 
+def validate_calico_network_cidr(calico_network_cidr, enable_windows_agents):
+    if enable_windows_agents.lower() == 'true':
```

Comment:
@jongiddy , I need your confirmation here.
The background is that we do not support calico in windows, so we don't need to validate calico configurations if DC/OS is deployed on windows.
- Can we just verify DC/OS on windows by `enable_windows_agents`?
- Is it possible to upgrade a dc/os cluster to support windows agents from just Linux agents?

Thanks in advance.

Reply:
CC @DominikDary

Reply:
1) yes
2) no